src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -151,6 +151,55 @@ void CascadiaSettings::_CreateDefaultProfiles()
     _profiles.emplace_back(powershellProfile);
     _profiles.emplace_back(cmdProfile);
     
+    // Add all WSL distros from: HKCU\Software\Microsoft\Windows\CurrentVersion\Lxss
+    Profile WSLDistro{};
+    HKEY hLXSSKey;
+    // Should I use RegOpenCurrentUser to access the uncached profile?
+    // https://docs.microsoft.com/en-us/windows/desktop/api/winreg/nf-winreg-regopencurrentuser#remarks
+    if (RegOpenKeyExW(HKEY_CURRENT_USER, L"Software\\Microsoft\\Windows\\CurrentVersion\\Lxss",
+        0, KEY_READ, &hLXSSKey) == ERROR_SUCCESS)
+    {
+        std::filesystem::path AppsDirectory = ExpandEnvironmentVariableString(L"%ProgramFiles%");

```

Comment:
Here and in other places where combined declaration and assignment in the new code is performed:
[ES.23: Prefer the {}-initializer syntax](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax)

Reply:
Thank you for taking the time to review my code and provide references for the corrections, it's been very educational.